### DISCUSSION TITLE
Collaboration support

### DISCUSSION THREAD

**Discussion by bengimizrahi** [jrnl-org/jrnl#1386] [Q&A]
Hi,

Would it be possible to maintain the journal.txt in a network file system and collaboratively maintain it between project members? Will there be any synchronization issues?

**Comment by micahellison** [comment]
Hi, unfortunately, there will be sync issues if multiple users are writing to the same journal. You could set up a different journal for each user, though I understand that this isn't the most convenient approach.

When bugs #1106 and #1107 are resolved, though, then it should be possible to have multiple users working on the same journal, but it could be some time until then.
